doc/benchmarking.md

```diff
@@ -2,54 +2,47 @@ Benchmarking
 ============
 
 Bitcoin Core has an internal benchmarking framework, with benchmarks
-for cryptographic algorithms (e.g. SHA1, SHA256, SHA512, RIPEMD160), as well as the rolling bloom filter.
+for cryptographic algorithms (e.g. SHA1, SHA256, SHA512, RIPEMD160, Poly1305, ChaCha20), rolling bloom filter, coins selection,
+thread queue, wallet balance.
 
 Running
 ---------------------
+
+For benchmarks purposes you only need to compile `bitcoin_bench`. Beware of configuring without `--enable-debug` as this would impact
+benchmarking by unlatching log printers and lock analysis.
+
+    make -C src bench_bitcoin
```

Comment:
Shouldn't this be `make -C src bench/bench_bitcoin` ? 
I think that would be clearer given all the following commands all use `src/bench/bench_bitcoin`

Reply:
Well in fact no.  I was surprised but `make -C src bench/bench_bitcoin` doesn't work. Maybe makefile target needs to be updated somewhere..

Reply:
That’s confusing, it worked for me when I tested it earlier?

Reply:
Dunno but I tested it again on my Debian and doesn't work. Should some configure flags be set correctly ?

Reply:
I'm just doing:
```bash
cd bitcoin
./autogen.sh
./configure
make -C src bench/bench_bitcoin
src/bench/bench_bitcoin --help
```

Reply:
Tried same and got a classic makefile error:

```
make: Entering directory '/home/user/Bitcoin/bitcoin/src'
make: *** No rule to make target 'bench/bitcoin_bench'.  Stop.
make: Leaving directory '/home/user/Bitcoin/bitcoin/src

Reply:
```sh
$ make -j 9 -C src bench/bench_bitcoin
make: Entering directory '/home/marco/workspace/btc_bitcoin_core/src'
make[1]: Entering directory '/home/marco/workspace/btc_bitcoin_core'
make[1]: Leaving directory '/home/marco/workspace/btc_bitcoin_core'
make: Leaving directory '/home/marco/workspace/btc_bitcoin_core/src'

$ make -j 9 -C src bench_bitcoin
make: Entering directory '/home/marco/workspace/btc_bitcoin_core/src'
make: *** No rule to make target 'bench_bitcoin'.  Stop.
make: Leaving directory '/home/marco/workspace/btc_bitcoin_core/src'

Reply:
What version of debian are you using?

Reply:
```
make -j 9 -C src bench_bitcoin
src/bench/bench_bitcoin --help
```
from root this works for me on Debian 4.19.37-5 (2019-06-19) x86_64.

Reply:
Debian 9.9

Reply:
Huh, in a fresh docker of debian 9.9 I get:


```
# lsb_release -d && make -j 16 -C src bench_bitcoin
Description:	Debian GNU/Linux 9.9 (stretch)
make: Entering directory '/bitcoin/src'
make: *** No rule to make target 'bench_bitcoin'.  Stop.
make: Leaving directory '/bitcoin/src'